shared/engine/index.platform.desktop.js

```diff
@@ -6,6 +6,7 @@ import {socketPath} from '../constants/platform.specific.desktop'
 
 class NativeTransport extends TransportShared {
   constructor (incomingRPCCallback, connectCallback) {
+    console.log('Transport using', socketPath)
```

Comment:
Ok LGTM!

On Thursday, September 1, 2016, Gabriel Handford <notifications@github.com>
wrote:

> In shared/engine/index.platform.desktop.js
> <https://github.com/keybase/client/pull/3962#discussion_r77244201>:
>
> > @@ -6,6 +6,7 @@ import {socketPath} from '../constants/platform.specific.desktop'
> >
> >  class NativeTransport extends TransportShared {
> >    constructor (incomingRPCCallback, connectCallback) {
> > +    console.log('Transport using', socketPath)
>
> Intentional.. good to have socket path in the log for debugging
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3962/files/6dc9af82922b09586b91ad9f6dae044623c26f0e#r77244201>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA05_57on_H55AyZMUlo0ZF7SmJNGCtEks5qlzDNgaJpZM4JoAqO>
> .
>